src/deluge/model/clip/instrument_clip.cpp

```diff
@@ -873,19 +873,14 @@ void InstrumentClip::processCurrentPos(ModelStackWithTimelineCounter* modelStack
 			// if probably setting has resulted in a note on
 			if (conditionPassed) [[likely]] {
 				// now we check if we should skip note based on iteration condition
-				int32_t iterance = pendingNoteOnList.pendingNoteOns[i].iterance & 127;
+				int32_t iterance = pendingNoteOnList.pendingNoteOns[i].iterance & 32767;
```

Comment:
it would be good to name this magic number something more meaningful

Reply:
Maybe after all we don't need the & 32767 part as we already have the source parameter as uint16 so when converted to int32 it appends zeroes anyway